lib/Runtime/Language/ModuleNamespace.h

```diff
@@ -20,7 +20,11 @@ namespace Js
     public:
         static ModuleNamespace* GetModuleNamespace(ModuleRecordBase* moduleRecord);
 
-        static bool Is(Var ns) { return ModuleNamespace::Is(ns); }
+        static bool Is(Var ns)
+        {
+            // This function calls itself ?
+            return false; // ModuleNamespace::Is(ns);
```

Comment:
I changed the body of this function to 

```c++
            AssertMsg(false, "ModuleNamespace::Is");
            return false; // ModuleNamespace::Is(ns);
```

Observed no failures when running the unit test suite.

I surmise that this function is likely not called under normal circumstances. I'm not necessarily sure that returning `false` is the right thing to do, but it's better than letting it pass through to DynamicObject::Is which could return `true` even if `ModuleNamespace::Is` should be false.

Maybe it's worth making this change and seeing how a Crawler run goes.